lib/src/bluez_client.dart

```diff
@@ -1292,7 +1529,7 @@ class BlueZClient {
 
   /// Creates a new BlueZ client. If [bus] is provided connect to the given D-Bus server.
   BlueZClient({DBusClient? bus})
-      : _bus = bus ?? DBusClient.system(),
+      : _bus = bus ?? DBusClient.system(introspectable: true),
```

Comment:
Can you put this in a separate PR.

Reply:
Leftover from some skill issue with trying to get this PR to work, unnecessary change and I'll remove it.

Reply:
Removed.